Makefile

```diff
@@ -0,0 +1,67 @@
+.PHONY: start, start_dev, register_hosts, run_api_faker, install_dependencies, save_dependencies, restore_dependencies
+
+define HOSTS
+127.0.0.1 api.twilio.com
+127.0.0.1 chat.twilio.com
+127.0.0.1 fax.twilio.com
+127.0.0.1 ip-messaging.twilio.com
+127.0.0.1 lookups.twilio.com
+127.0.0.1 messaging.twilio.com
+127.0.0.1 monitor.twilio.com
+127.0.0.1 notifications.twilio.com
+127.0.0.1 notify.twilio.com
+127.0.0.1 preview.twilio.com
+127.0.0.1 pricing.twilio.com
+127.0.0.1 sync.twilio.com
+127.0.0.1 taskrouter.twilio.com
+127.0.0.1 video.twilio.com
```

Comment:
@joliveros is there a way to move these paths to a file and read the content of the file instead of defining it inside the Makefile?
In case someone needs to add a new path, the commit will show the git-diff within the file `hosts` (assuming that's the name of the file) and that's clear to read ^^

Reply:
that makes sense. I'll see if I can figure it out :)